stix2elevator/convert_stix.py

```diff
@@ -903,8 +903,15 @@ def add_relationships_to_reports(bundle_instance):
             rep["object_refs"] = rels_to_include
 
 
+def handle_missing_required_property(property_name, instance):
+    if get_option_value("ignore_required_properties"):
+        warn("Required property '%s' is not populated on %s", 437, property_name, instance["id"])
+    else:
+        warn("A placeholder was generated for required property '%s' of %s", 438, property_name, instance["id"])
```

Comment:
should this function set the placeholder value too (not just warn) or did I miss where that happens?

Reply:
Good catch - I will fix